modules/desktop/electron/electron.ts

```diff
@@ -38,11 +38,10 @@ if (app.isPackaged) {
     }
   });
   Sentry.configureScope(async (scope) => {
-    const [session, cliVersion] = await Promise.all([readSessionData(), initializeTeaCli()]);
+    const session = await readSessionData();
     scope.setUser({
       id: session.device_id, // device_id this should exist in our pg db: developer_id is to many device_id
-      username: session?.user?.login || "", // github username or handler
-      tea: cliVersion
```

Comment:
@ABevier we would need to track the teaxyz/lib version.

Reply:
I'm not sure I understand why this is necessary any more?  teaxyz/lib is a static dependency of the gui now.  We know the version just by looking at the gui version and then looking at what that version of the GUI was built with.